DirectProgramming/C++SYCL_FPGA/Tutorials/Features/lsu_control/src/lsu_control.cpp

```diff
@@ -81,7 +81,7 @@ void KernelRun(const std::vector<int> &input_data, const size_t &input_size,
       h.single_task<KernelPrefetch>([=]() [[intel::kernel_args_restrict]] {
         auto input_ptr =
             input_a.template get_multi_ptr<access::decorated::no>();
-        auto output_ptr = output_a.get_pointer();
+        auto output_ptr = output_a.get__multi_pointer();
```

Comment:
Is the `__` intended?

Reply:
It is not. This should be corrected now.